setup.py

```diff
@@ -33,6 +33,7 @@
 
     install_requires=[
         "PyCap>=2.1.0"
```

Comment:
needs a comma i think

Reply:
This was one of the files which never got committed, so this shouldn't have been here at the first place. This fixes python_db_logger not getting installed when you do a `pip install .`